it/test/uk/gov/hmrc/uknwauthcheckerapi/BaseISpec.scala

```diff
@@ -44,13 +43,17 @@ class BaseISpec extends PlaySpec with GuiceOneServerPerSuite with WireMockISpec
     TestConstants.serviceNameAuth,
     TestConstants.serviceNameIntegrationFramework
   )
+
   override lazy val app: Application = GuiceApplicationBuilder()
     .configure(additionalAppConfig)
     .build()
+
   protected lazy val authorisationsUrl = s"http://localhost:$port/authorisations"
   protected lazy val appConfig: AppConfig = injected[AppConfig]
   private lazy val wsClient:    WSClient  = injected[WSClient]
 
+  override protected lazy val minMaxValues: MinMaxValues = injected[MinMaxValues]
```

Comment:
I don't see a point in this overriding a base value. MinMaxValues is a class, and TestData is generated data. We're not generating MinMaxValues, so we can just define it in the BaseISpec if we need it.